### PR TITLE
NXP Electric Vehicle Source Equipment (EVSE) Plug and Play Model

### PR DESCRIPTION
**Pull request by Swinny619**
### Thank you for contributing to the Azure IoT Plug and Play Models repository

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

The repository pipeline will ensure minimum validation of incoming DTDL models.  

- PR validation steps are described in the tools [Wiki](https://github.com/Azure/iot-plugandplay-models-tools/wiki/Validation-Pipeline#pr-validation-checks).
- The same validation can be run locally via the [dmr-client](https://github.com/Azure/iot-plugandplay-models-tools/tree/dev/clients/dotnet#device-model-repository-client) CLI.

## Requested info template for model(s) submission

When submitting models to the repository we ask that you provide as much of the following meta information around your models and related devices as possible. This info will be used to improve Plug and Play.

:star2: Please replace the markdown comment examples with your own values.

### Company Info

- NXP Semiconductors
- nxp.com

-->

### CODE REVIEW

File: dtmi/nxp/evse-1.json [hunk 1]
```diff
@@ -0,0 +1,244 @@
+[
```
Comment (rido-min) [inline dtmi/nxp/evse-1.json:1]: We require to describe a single interface per file, hence we do not support an Array.

**Review by rido-min — CHANGES_REQUESTED** [review]
Remove the `[` `]`

File: dtmi/nxp/evse-1.json (hunk 1 above)
Comment (Swinny619) [reply dtmi/nxp/evse-1.json]: I Have removed the [ and ] and updated .json.  Interesting that the original model file past the validation checks.

Comment (rido-min) [reply dtmi/nxp/evse-1.json]: DTDL allows an Array as a top level element, however the device model repository has stricter rules, such as one interface per file as described [here](https://github.com/Azure/iot-plugandplay-models/blob/main/pr-reqs.md#model-file-content-structure).

If you are using the `dmr-client validate` command, make sure to use the `--strict` option to verify this case.

**Review by rido-min — APPROVED** [review]

**Pull request merged**